.travis.yml

```diff
@@ -7,6 +7,7 @@ notifications:
   email: false
 node_js:
   - 'node'
+  - '7'
```

Comment:
Since we're only running it on node 4+, I'm not really sure (npm 4.6+ breaks on < v1, and v5+ breaks on < v4).

It might be a bug with xo?